### DISCUSSION TITLE
Change how Channels are Identified by Country

### DISCUSSION THREAD

**Discussion by MapGuy11** [iptv-org/community#115] [Ideas]
Hello All!

I propose a change to the system we currently have, which is where channels are sorted by the home countries network. For example, Disney in the UK is a US flag due to the fact that Disney is based in the US. The issue with this is Disney UK doesn't have the same owners as Disney US though may have the same content and logos. It honestly creates confusion with the viewer because if they live in the UK and would want Disney Channel most likely they would look first in the UK.m3u file instead of the us one. While most players have a search function and categories I feel that it would be easier to add channels and when addressing issues or pull requests to go by the audience country instead of the networks home country.

Thanks please put your thoughts below.

**Comment by BellezaEmporium** [comment]
Changing things so late would create a total chaos in some of the playlists we got lol.

What's going to be difficult is for the "Latinoamérica" channels, because we cannot trace from where they originate, and it's going to be another pair of gloves to correctly find them all.

**Comment by MapGuy11** [comment]
> Changing things so late would create a total chaos in some of the playlists we got lol.
> 
> What's going to be difficult is for the "Latinoamérica" channels, because we cannot trace from where they originate, and it's going to be another pair of gloves to correctly find them all.

I mean we could start with the blatantly obvious ones like Disney UK in the US playlist. I don't know just a thought.

**Comment by freearhey** [comment]
Before we can do this, we need to update the description of channels in the [@iptv-org/database](https://github.com/iptv-org/database) so that all contributors can also quickly identify the country.

I myself have already started to do this, but as @LaneSh4d0w correctly pointed out this process is not simple and will take a long time.

As for the definition of the country for channels like "Disney Channel Latin America", I personally use the country listed in the Wikipedia or the address of the company's headquarters.
